pkg/distributors/test/Reinvestor.test.ts

```diff
@@ -114,7 +114,7 @@ describe('Reinvestor', () => {
         ).wait();
 
         const deltas = [bn(0), bn(0)];
-        deltas[assets.indexOf(rewardToken.address)] = bn('999999999999999898');
+        deltas[assets.indexOf(rewardToken.address)] = bn('999999999999999498');
```

Comment:
Is this value different because we're advancing a different amount of time? Can this be computed instead of hardcoded?